src/main/java/org/springframework/data/aerospike/core/AerospikeOperations.java

```diff
@@ -66,18 +66,11 @@
 	MappingContext<?, ?> getMappingContext();
 	
 	/**
-	 * Save operation using the WritePolicy.recordExisits policy of CREATE_ONLY
+	 * Save operation
 	 * @param objectToInsert
 	 */
 	void save(Object objectToInsert);
 	
-	/**
-	 * Save operation using the WritePolicy specified.
-	 * @param objectToInsert
-	 * @param policy
-	 */
-	void save(Object objectToInsert, WritePolicy policy);
-	
```

Comment:
Thanks for the clarification.  The comments over "save" method may need some tune-up:
	 * Save operation using the WritePolicy.recordExisits policy of CREATE_ONLY		 +	 * Save operation
  
See your point and the logic from couchbase in your update.  It sounds like a very good idea to make all those option/policy configurable if that is what you are pursuing.